File: lesson4/c_c++/lesson4_11.cpp
#include <iostream>

int main() {
    int32_t numb;

    for (numb = 2; numb < 100; numb++) {

        int32_t k_numb = 2;
        while (k_numb * k_numb <= numb and numb % k_numb != 0) {
            k_numb += 1;
        }
        if (k_numb * k_numb > numb) {
            std::cout << "numb = " << numb << std::endl;
        }
    }

    return 0;
}

File: lesson4/python3/lesson4_11.py
#!/usr/bin/python3


for numb in range(2, 101):

    k_numb = 2
    while k_numb * k_numb <= numb and numb % k_numb != 0:
        k_numb += 1
    if k_numb * k_numb > numb:
        print("numb =", numb)

